src/components/Table/HeaderCell.tsx

```diff
@@ -0,0 +1,77 @@
+import * as React from 'react'
+
+import Icon from '../Icon'
+import { SortableCellUI, HeaderCellUI } from './styles/Table.css'
+import { TABLE_CLASSNAME } from './Table'
+
+import { HeaderCellProps } from './types'
+
+export default class HeaderCell extends React.PureComponent<HeaderCellProps> {
+  render() {
+    const { column, sortedInfo } = this.props
+
+    return (
+      <HeaderCellUI
+        className={`${TABLE_CLASSNAME}__HeaderCell`}
+        align={column.align}
+        cellWidth={column.width}
+        aria-sort={
+          sortedInfo &&
+          sortedInfo.columnKey === column.columnKey &&
+          sortedInfo.order
```

Comment:
For long and/or non-obvious conditions like this it would be great to break it out into a variable to help some reading the code so that they don't have to interpret it. I believe what's meant here could be expressed as:

```
const isSortedBy = sortedInfo && sortedInfo.columnKey === column.columnKey && sortedInfo.order
...
aria-sort={isSortedBy ? sortedInfo.order : 'none'}
```

Not a blocker though at all.

Reply:
Went ahead and refactored that bit, thanks for the suggestion!